lib/internal/http2/core.js

```diff
@@ -2731,11 +2730,9 @@ function connect(authority, options, listener) {
 // Support util.promisify
 Object.defineProperty(connect, promisify.custom, {
   value: (authority, options) => {
-    const promise = createPromise();
-    const server = connect(authority,
```

Comment:
Actually, it seems we might be able to remove the custom promisifier entirely: what the getter is doing right now should be done by `util.promisify`.

Reply:
A test fails if that part is removed.